### PR TITLE
Fixes #471 Move API documentation to docs folder

### LINKED ISSUE #471: Move API documentation to docs folder
The API documentation is still on the site itself here http://api.susi.ai/api.html. Please move the documentation to the docs folder and link it from the Readme.md.

### PR DESCRIPTION
**Pull request by manishv123**
Fixes #471 

Changes: API Documentation was added in the docs folder from the link http://api.susi.ai/api.html and a link was provided from readme.rst file to access that documentation.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -185,9 +185,7 @@ http://loklak.org/api/search.rss?q=0rb1t3r or just follow @0rb1t3r
 Where can I find documentation?
 -------------------------------
 
-The application has built-in documentation web pages, you will see them
-when you opened the application web pages or you can simply open
-``html/index.html`` or just use http://api.susi.ai as reference.
+The Doumentation can be found at https://github.com/manishv123/susi_server/blob/development/docs/API_Documentation/API_Documentation.md
```
Comment (mariobehling) [inline README.rst]: why your personal repo?

**Comment by manishv123** [comment]
@mariobehling please review again i have made the suggested changes.

**Comment by manishv123** [comment]
@DravitLochan can u help why i am seeing others commits in my pull request and i think because of this i am having error in squashing commits?

**Comment by DravitLochan** [comment]
@manishv123 Yes. I can see there are previous commits in your pr, not made by you. I would advise to cherry pick the commits and squash them.

**Pull request closed without merging**